packages/lambda-at-edge/src/default-handler.ts

```diff
@@ -24,6 +26,11 @@ const addS3HostHeader = (
 
 const isDataRequest = (uri: string): boolean => uri.startsWith("/_next/data");
 
+const normaliseUri = (uri: string): string => {
+  if (basePath && uri.startsWith(basePath)) uri = uri.slice(basePath.length);
+  return uri === "/" || uri === "" ? "/index" : uri;
```

Comment:
The `/` to `/index` normalisation isn't necessary anymore. I think you can just return `uri` directly after the basePath normalisation;

```suggestion
  return uri;
```

Reply:
Removing `uri === "/"` works. If I remove `uri === ""` then this specific test fails: https://github.com/serverless-nextjs/serverless-next.js/pull/510/files#diff-1e2c3ac8a5020d5e147d22a7d7df9029R49

Reply:
What if you do `return uri === "" ? "/" : uri;` ?